app/src/main/java/com/github/h3lp3rs/h3lp/HelpParametersActivity.kt

```diff
@@ -89,11 +102,24 @@ class HelpParametersActivity : AppCompatActivity() {
             b.putStringArrayList(EXTRA_NEEDED_MEDICATION, meds)
             val intent = Intent(this, AwaitHelpActivity::class.java)
             intent.putExtras(b)
-
+            sendInfoToDB()
             startActivity(intent)
         }
     }
 
+    /**
+     * Stores the emergency information in the database for further use
+     */
+    private fun sendInfoToDB(){
+        if(latitude != null && longitude != null){
+            val medicalInfo = getSharedPreferences(getString(R.string.medical_info_prefs), MODE_PRIVATE).getString(getString(R.string.medical_info_key),"")
+            val gson = Gson()
+            val medicalInfoJson = gson.toJson(medicalInfo)
+            val emergencyInfo = EmergencyInformation(latitude = latitude!!, longitude = longitude!!, meds =  meds, time = currentTime, medicalInfo = medicalInfoJson)
+            emergencyInfoRepository.create(emergencyInfo)
+        }
```

Comment:
What happens if the latitude or longitude is null? Do we consider that if one of the two is missing that the information should simply not be sent to the database, which does make sense because otherwise, it is just simply not possible to provide this person with help if the person willing to help can't localize them. However, it would be wise to give feedback to the user stating that we were not able to access his location and therefore can't broadcast his request for help. Either send a reminder that he should accept to share his location, as is done in most applications nowadays, if that was the issue, or try to access the location again if the issue was coming from something else. Anyways, this is not something you have to fix, we should discuss this as a group.